### DISCUSSION TITLE
KEDA pod scaler using metrics-api and down scaling question. edit: answer found

### DISCUSSION THREAD

**Discussion by mikarinneoracle** [kedacore/keda#4323] [Q&A / Need Help] [answered]
Hi 

edit: I found the answer for long running jobs and scaling them using Keda
https://keda.sh/docs/2.9/concepts/scaling-jobs/

Many thanks!

**Comment by tomkerkhove** [comment, marked as answer]
Looks like this is resolved, good to hear!

**Reply by mikarinneoracle** [reply]
Thank you, yes it is :) Do you know if Keda job scaler can also scale on Node level or does that have to be done in combination with another Node level autoscaler (like monitoring node CPU usage for instance) ? Thanks again!

**Reply by tomkerkhove** [reply]
Nope, KEDA only scales applications.

**Reply by tomkerkhove** [reply]
If the node CPU information is available in one of the scalers you can use it but we don't scale nodes unless another technology provides a CRD that KEDA can scale

**Reply by mikarinneoracle** [reply]
Thanks you, good to know (I expected so but wanted to check). Thanks again!

**Reply by mikarinneoracle** [reply]
Just wanted confirm this works great with OCI queue and metrics-api (for NodeJS) :)
